### PR TITLE
Update rules-dsl.md

### PR DESCRIPTION
**Pull request by Rossko57**
Added some Quantity handling details to "Number Item" to give hints for perennial comparison and temperature addition problems

Signed-off-by: Ross Kennedy rossko@culzean.clara.co.uk

### CODE REVIEW

File: configuration/rules-dsl.md [hunk 1]
```diff
@@ -609,6 +617,29 @@ One warning comes with DecimalType (also holds true for QuantityType).
 The full explanation is [beyond the scope of this introduction](https://community.openhab.org/t/ambiguous-feature-call-whats-wrong-designer-user-or-bug/9477/4).
 To avoid an error mentioning an "Ambiguous Method Call" always cast the state of a DecimalType to a Number, not DecimalType.
 
+Take care with maths around Quantity Types.  While you can freely mix units in many cases, there are pitfalls.
+```java
```
Comment (Confectrician) [inline configuration/rules-dsl.md:621]:
```suggestion

```java
```

**Review by Confectrician — COMMENTED** [review]
Hi,

Just one small comment, to keep the markdown correct.

File: configuration/rules-dsl.md (hunk 1 above)
Comment (Rossko57) [reply configuration/rules-dsl.md:622]: Thanks, made change suggested

**Comment by Rossko57** [comment]
Thanks, made change suggested

**Review by Confectrician — APPROVED** [review]
Thanks 👍

No [sign-off](https://github.com/openhab/openhab-docs/blob/master/CONTRIBUTING.md#sign-your-work) needed due to [small patch exception](https://github.com/openhab/openhab-docs/blob/master/CONTRIBUTING.md#small-patch-exception)

**Pull request merged**